dvc/objects/errors.py

```diff
@@ -1,15 +1,8 @@
-from dvc.exceptions import DvcException
-
-
-class ObjectError(DvcException):
-    pass
-
-
-class ObjectFormatError(ObjectError):
+class ObjectFormatError(Exception):
```

Comment:
I am trying to create a hierarchy of exception classes. I think we should try to have a hierarchy when the exceptions are related, but otherwise avoid base exceptions. This forces us to be explicit about catching related exceptions instead of some common abstract exceptions (eg: `DvcObjectsBaseError`, etc.)

See https://github.com/jelmer/dulwich/blob/master/dulwich/errors.py.